ethereum/ethereum-reth/main.rhai

```diff
@@ -0,0 +1,112 @@
+import "base" as base;
+import "reth" as reth;
+
+const CADDY_DIR = DATA_DRIVE_MOUNT_POINT + "/caddy";
+const RPC_PORT = 8545;
+const WS_PORT = 8546;
+const METRICS_PORT = 6665;
+
+let RETH_CONFIG = reth::base_config(global::METRICS_PORT,global::RPC_PORT,global::WS_PORT,global::CADDY_DIR);
+
+const BASE_CONFIG  = #{
+    config_files: RETH_CONFIG.config_files+base::BASE_CONFIG.config_files,
+    services: RETH_CONFIG.services+base::BASE_CONFIG.services,
+};
+
+const VARIANTS = #{
+    reth_sepolia_full: #{
+        net: "sepolia",
+    },
+    reth_mainnet_full: #{
+        net: "mainnet",
+    },
+    reth_sepolia_archive: #{
+        net: "sepolia",
+    },
+    reth_mainnet_archive: #{
+        net: "mainnet",
+    },
+};
+
+const API_HOST = `http://127.0.0.1:${global::RPC_PORT}`;
+const HOME = PROTOCOL_DATA_PATH + "/reth";
+const NET = VARIANTS[node_env().node_variant].net;
+const CADDY_DIR = DATA_DRIVE_MOUNT_POINT + "/caddy";
+
+const RETH_EXTRA_ARGS = if node_params().nodetype == "full" {
+    "--db-mode=full --pruning.block-limit=1048576"
+} else {
+    "--db-mode=archive"
+};
+
+const BABEL_VERSION = "0.9.0";
+
+const PLUGIN_CONFIG = #{
+    init: #{
+        commands: [
+            `mkdir -p ${global::HOME}`,
+            `mkdir -p ${global::CADDY_DIR}`,
+        ],
+        jobs: [
+            #{
+                name: "create_jwt",
+                run_sh: `export HOME=/root && openssl rand -hex 32 | tr -d "\n" > ${global::HOME}/jwt.hex`,
+            }
+        ]
+    },
+    services: [
+        #{
+            name: "reth",
+            run_sh: `export HOME=/root && /root/bin/reth node \
+                    --datadir=${global::HOME} \
+                    --chain=${global::NET} \
+                    --http \
+                    --http.addr=127.0.0.1 \
+                    --http.port=${global::RPC_PORT} \
+                    --http.api=eth,net,web3,debug,trace,txpool \
```

Comment:
We defined tracing in the babel.yaml file as a configurable value but then it's hardcoded here.